### PR TITLE
Add new screen reader texts for AddressSearchBar when cleared and extra info

### PR DESCRIPTION
**Pull request by Dremars**

### CODE REVIEW

**Comment by Dremars** [comment]
- Missing translations.
- Need to investigate focus problem with newer devices. Problem persist on Android v7 even with manual focus back to input. Need to check newer Androids and iPhone

File: src/components/AddressSearchBar/AddressSearchBar.js [hunk 1]
```diff
@@ -93,7 +98,23 @@ const AddressSearchBar = ({
...
+    infoText = <FormattedMessage id="address.search.cleared" />;
+  } else {
+    infoText = (
+      <>
+        {infoText}
+        {
+          addressResults.length
+          && (
+            <FormattedMessage id="address.search.suggestion" />
+          )
```
Comment (mikkoisa) [inline src/components/AddressSearchBar/AddressSearchBar.js:112]: For some reason React displays "0" here if no address results. Works with:
` addressResults.length
            ? (
              <FormattedMessage id="address.search.suggestion" />
            ) : null`

**Review by mikkoisa — CHANGES_REQUESTED** [review]

**Comment by Dremars** [comment]
[(3) Kotiosoite-kentän korjaukset](https://trello.com/c/3fJ4BfEq/740-3-kotiosoite-kent%C3%A4n-korjaukset)

File: src/components/AddressSearchBar/AddressSearchBar.js (hunk 1 above)
Comment (Dremars) [reply src/components/AddressSearchBar/AddressSearchBar.js]: Should be fixed now (bef2efa)

**Comment by Dremars** [comment]
@mikkoisa Usability is still horrible on Android v7 but it seems to be inherent issue with that version. Can you check if all works fine on iPhone. I'll check Android v10 when I get a chance.

**Comment by Dremars** [comment]
Seems to work pretty well on Android v10

**Review by mikkoisa — APPROVED** [review]
Works on iOS as well

**Comment by Dremars** [comment]
Translations still missing.

**Pull request merged**